qutip/bloch3d.py

```diff
@@ -50,15 +50,15 @@ class Bloch3d():
         Width of displayed vectors.
     view : list {[45,65]}
         Azimuthal and Elevation viewing angles.
-    xlabel : list {['|x>', '']}
+    xlabel : list {``['|x>', '']``}
```

Comment:
What are the extra quotes here for? I can imagine strange formatting is needed, but I don' know exactly why in this case. :)

Reply:
> is a special character in .rst, and since the docs weren't able to build I tried changing adding the backticks as @BoxiLi suggested in order to consider it a regular character in a string. I still wasn't able to entirely build the docs for the Bloch3d class locally, so we decided to remove the Bloch3d class from the docs for now. The html was partially built though including the part with these statements, so I was able to check whether the backticks affected anything, which they did: the statement was changed from a hyperlink (that led nowhere) to a regular string. So I decided to keep them even though the Bloch3d class wasn't added to the rst file.

Reply:
Thanks!